Mail/Views/Settings/General/SettingsNotificationsView.swift

```diff
@@ -49,8 +50,11 @@ struct SettingsNotificationsView: View {
                             .textStyle(.bodySecondary)
                             .frame(maxWidth: .infinity, alignment: .leading)
                         Button(MailResourcesStrings.Localizable.warningNotificationsDisabledButton) {
-                            guard let settingsUrl = URL(string: UIApplication.openSettingsURLString) else { return }
-                            openURL(settingsUrl)
+                            let settingsUrl = platformDetector
+                                .isMac ? "x-apple.systempreferences:com.apple.preference.notifications" : UIApplication
```

Comment:
You can create a constant for this URL in the enum `DeeplinkConstants`.
Something like `macNotifications`